src/main/java/org/jfree/data/xy/XYShapeDataItem.java

```diff
@@ -0,0 +1,66 @@
+package org.jfree.data.xy;
+
+import java.awt.Shape;
+import java.io.Serializable;
+
+public class XYShapeDataItem extends XYDataItem implements Cloneable, Serializable{
+
+	
+	private static final long serialVersionUID = 2477972108001149458L;
+
+	private Shape shape;
+	private Boolean hasShape = false;
+	public XYShapeDataItem(double x, double y) {
+		super(x, y);
+	}
+	
+	public XYShapeDataItem(Number x, Number y) {
+        super(x, y);
+    }
+	
+	public XYShapeDataItem(double x, double y, Shape shape) {
+		super(x, y);
+		// TODO Auto-generated constructor stub
```

Comment:
Serait-ce possible d'implémenter le stub pour cette itération?

Reply:
Elle est implémenté, seulement un commentaire inutile...